vendor/modules.txt

```diff
@@ -2189,6 +2189,7 @@ k8s.io/kubectl/pkg/cmd/wait
 k8s.io/kubectl/pkg/describe
 k8s.io/kubectl/pkg/drain
 k8s.io/kubectl/pkg/explain
+k8s.io/kubectl/pkg/explain/v2
```

Comment:
We really shouldn't have to get approval for that, especially when this is probably checked by CI, it's not like there's anything needed here.